src/pages/search.tsx

```diff
@@ -1,67 +1,157 @@
-  }
+
+  const { data: responseSkillsDetails, error: skillsDetailsError } =
+    useQuery<GetUserDesireOnEachSkillQuery>(GET_USER_DESIRE_ON_EACH_SKILL, {
+      variables: { search: `%${search}%` },
+    });
+
+  if (error) console.error(error);
+
+  if (skillsDetailsError) console.error(skillsDetailsError);
+
   const skills = data?.skills;
+  const skillsDetails = responseSkillsDetails?.Skill;
   const profiles = data?.profiles;
 
+  const sortedSkills = () => {
+    if (skills && skillsDetails) {
+      const xSkills = [...skills];
+      const skillsDetailsSorted = [...skillsDetails];
+      switch (filter.label) {
+        case "default":
+          if (search === "") {
+            return xSkills.slice(0, 10);
+          }
+          return skills;
+        case "trendsAsc":
+          if (search === "") {
+            return xSkills
+              .sort(
+                (a, b) =>
+                  Number(b.desireLevel + b.skillLevel) / 2 -
+                  Number(a.desireLevel + a.skillLevel) / 2
+              )
+              .slice(0, 10);
+          }
+          return xSkills.sort(
+            (a, b) =>
+              Number(b.desireLevel + b.skillLevel) / 2 -
+              Number(a.desireLevel + a.skillLevel) / 2
+          );
+        case "mostNoted":
+          if (search === "") {
+            return skillsDetailsSorted
+              .sort(
+                (a, b) => b.UserSkillDesires.length - a.UserSkillDesires.length
+              )
+              .slice(0, 10);
+          }
+          return skillsDetailsSorted.sort(
+            (a, b) => b.UserSkillDesires.length - a.UserSkillDesires.length
+          );
+        default:
+          return skills;
+      }
+    } else {
+      return [""];
+    }
+  };
+
   return (
-    <PageWithNavAndPanel pathName={pathName} context={""}>
-      <div className="flex justify-center mb-16">
-        <div
-          className={`flex ${isDesktop ? "w-2/3" : "w-full"}  flex-col mx-4 `}
-        >
-          <SearchBar
-            initialValue={search}
-            value={search}
-            setSearch={setSearch}
-            placeholder={t("search.placeholder")}
-          />
-          {search && search !== "" ? (
-            <>
-              <div className="flex flex-col my-2 py-2">
-                <h1 className="text-xl">{t("search.skills")}</h1>
-                {skills?.length > 0 ? (
-                  skills.map((skill) => (
-                    <SkillPanel
-                      key={skill.name}
-                      skill={skill}
-                      categoryLabel={skill.Category?.label}
-                      context={"zenika"}
-                    />
-                  ))
-                ) : (
-                  <span className="text-sm">{t("search.noSkill")}</span>
-                )}
+    <>
+      <div className="bg-light-med dark:bg-dark-dark">
+        <PageWithNavAndPanel pathName={pathName} context={""}>
+          <></>
+        </PageWithNavAndPanel>
```

Comment:
```suggestion
        <PageWithNavAndPanel pathName={pathName} context={""}>
```

Reply:
Suggested this as we can all enclose in PageWithNavAndPanel component to maintain consistency between all pages

Reply:
I'm not sure if it's very usefull to all enclose in PageWithNavPanel. It creates incoherencies in css classes of the component

Reply:
My idea was to separate the search section of the result 
![Capture d’écran de 2022-09-20 10-47-15](https://user-images.githubusercontent.com/85115963/191212575-28a1e34b-8c91-4d3a-84e0-d6e7ae421aa4.png)

To get different sections to add maybe more filters in the future. (Such as https://dribbble.com/shots/16032266-himalayas-app-filtering-2-0) 

But if we enclose the component on the topBar, we can't reach the "width-full" and we are too dependent of the topBar css. And we can't take all the space of the page. I thinks it's too contraignant. 

Should we maybe revise the others component to not be dependant of the top bar ?

Reply:
Maybe we should fix this because it's weird to use a PageWithNavAndPanel component in this way, with only a React.Fragment component

The page `zenika` is working like this, that's why I thought it could be done

Reply:
Ok ! I see what you mean. It's also weird to have the search bar in the topbar (visually)

I think, we should revise these component but in another issue, it's not the main subject of this PR

Reply:
Yes okay. I'll re enclose the PageWithNavAndPanel in the component for this PR. We have to keep in mind to re order all of this :+1: